test/test_files.py

```diff
@@ -279,6 +279,16 @@ def test_copy_to_host(alpine_310):
     finally:
         os.remove(fn_host)
 
+    # sym-linked file
+    shell.run('ln -s /tmp/hello /tmp/linked')
+    _, fn_host = tempfile.mkstemp()
+    try:
+        files.copy_to_host('/tmp/linked', fn_host)
+        with open(fn_host, 'r') as f:
+            assert f.read() == (content + '\n')
+    finally:
+        os.remove(fn_host)
```

Comment:
Need to remember to destroy the symlink.

Reply:
I'm not sure if I correctly understand what you mean here. You mean removing the symlinked file on the container?

Reply:
Sorry. This was a very silly comment. I glanced at this and thought that the symlink was being created on the host. My bad! I'll go ahead and merge.